### PR TITLE
Change back ticks to double quotes

### PR DESCRIPTION
**Pull request by lherman-cs**
This change fixes the confusion between words referring to standard
prose and words referring not specific technical/informative term

Closes https://github.com/coala/coala-bears/issues/1851

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by mridubhatnagar** [comment]
Guide for writing good commit messages.  http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

**Comment by lherman-cs** [comment]
Oh, I thought the tag was just optional. Thank you for letting me know.

**Comment by mridubhatnagar** [comment]
@lherman-cs hey. You could have amended your commit message using the git command.
git commit --amend. 
But it seems you have made 3 different commits. By doing git commit. 
So, now you will have to squash your 3 commits into 1 commit.  
For squashing of commits. Follow our git basics guide. 
http://api.coala.io/en/latest/Developers/Git_Basics.html :)

**Comment by lherman-cs** [comment]
I've fixed the problem. I think it should be okay now. I'm sorry for all the troubles. I'm still a beginner. Please feel free what I'm supposed to do. Thanks.

**Comment by mridubhatnagar** [comment]
Also, before you push your changes run coala locally on your machine. It will tell you the errors if there are any. Resolve those and then push the changes. This way you won't be seeing the gitmate errors that are currently coming up.

**Comment by lherman-cs** [comment]
Actually, I ran coala before. But, it didn't tell me any error. Maybe, I ran it incorrectly. I'm not sure, to be honest. I'll research it later to make sure I did correctly.

**Comment by lherman-cs** [comment]
What should I do now? I don't know why the change gave me a command timed out. It seems unreasonable to me. Would you mind to help me? Thanks

**Comment by mridubhatnagar** [comment]
Can you please click on details icon mentioned corresponding to circle ci. And then you would see a restart build option. Can you please give this a try.

**Comment by lherman-cs** [comment]
I can't. I don't have the right permission. It says, "You need write permissions to trigger builds."

**Comment by mridubhatnagar** [comment]
Your fix seems to be alright. For checking why the Circle Ci is failing and giving a timeout error. You can ping in the gitter channel. 

And one last change in commit message. Instead of 

Change back ticks to double quotes.
This change fixes the confusion between words referring to standard
prose and words referring not specific technical/informative term.

Can you change it to 
Change back ticks to double quotes.This change fixes the 
confusion between words referring to standard prose 
and words referring not specific technical/informative term.

This looks much better.

**Comment by lherman-cs** [comment]
Sure, I'll do that.

**Comment by lherman-cs** [comment]
My commit failed again on continuous-integration/appveyor/pr. In the detail, it says the following: https://lh3.googleusercontent.com/tnCCxzrmTIfj9il7hTXLoVd2Iqob5nuUPJvvwYr8ln0w1b23jeopTfh7ZLTf9h02fecLwdLubcTsw1w=w1366-h649-rw

File: bears/natural_language/WriteGoodLintBear.py [hunk 1]
```diff
@@ -42,20 +42,20 @@ def create_arguments(filename, file, config_file,
                          allow_extra_words: bool=True,
                          allow_cliche_phrases: bool=True):
         """
-        Using ``False`` will enable the check.
+        Using "False" will enable the check.
```
Comment (jayvdb) [inline bears/natural_language/WriteGoodLintBear.py]: This PR is only about human language words in the parameter descriptions, not `False` here, which is a technical term.

File: bears/natural_language/WriteGoodLintBear.py [hunk 2]
```diff
@@ -42,20 +42,20 @@ def create_arguments(filename, file, config_file,
...
+                                        "very", "extremely", etc.
         :param allow_repeated_words:    Allows lexical illusions – cases
                                         where a word is repeated.
-        :param allow_there_is:          Allows ``There is`` or ``There are``
+        :param allow_there_is:          Allows "There is" or "There are"
                                         at the beginning of the sentence.
-        :param allow_ambiguous_words:   Allows ``weasel words`` for example
-                                        ``often``, ``probably``
+        :param allow_ambiguous_words:   Allows "weasel words" for example
+                                        "often", "probably"
```
Comment (jayvdb) [inline bears/natural_language/WriteGoodLintBear.py]: please add `and` after the last comma, and append a `.` (full stop) like the other descriptions.

**Comment by jayvdb** [comment]
Please rebase to fix the Circle CI build.

File: bears/natural_language/WriteGoodLintBear.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
         )
 class WriteGoodLintBear:
     """
-    Lints the text files using ``write-good`` for improving proses.
+    Lints the text files using "write-good" for improving proses.
```
Comment (jayvdb) [inline bears/natural_language/WriteGoodLintBear.py]: `write-good` is a technical term - it is the name of the program used.  It shouldnt be replaced with textual quotes

**Comment by rhemon** [comment]
LGTM

**Comment by rhemon** [comment]
ack dac2830

**Comment by sims1253** [comment]
ack dac28307fdb8c7864927260715614b5b20b92a3b

**Comment by sims1253** [comment]
reack 8973192d225af31baae357a0cc3f5ef7c138fd8a

**Comment by sims1253** [comment]
@rultor merge

**Pull request merged**